scripts/test_async_subagent_integration.py

```
@@ -0,0 +1,331 @@
+    uv run python scripts/test_async_subagent_integration.py [--verbose]
+"""
+
+import argparse
+import asyncio
+import sys
+from typing import List, Tuple
+
+# Add parent directory to path
+sys.path.insert(0, ".")
+
+from massgen.subagent.models import SubagentResult  # noqa: E402
```

Ruff (0.14.13)

Unused `noqa` directive (non-enabled: `E402`)

Remove unused `noqa` directive

(RUF100)